### PR TITLE
Configure redux store

### PR DESCRIPTION
**Pull request by Bigizi**
**Book-store project**
I added a redux repository with store js and 2 subfolders(Books and categories)
In the subfolder books, I added a Books.js file with an empty array book and two reducers(add and remove book)
In the subfolder categories, I added a categories.js file with an empty array and a reducer to check the status.

### CODE REVIEW

File: src/redux/books/Books.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+});
+
+const removeBook = (book) => ({
+  type: Removebook,
+  payload: book,
+});
+
+// Export
+export { addBook, removeBook };
+export default booksReducer;
```
Comment (TedLivist) [inline src/redux/books/Books.js]: - [ ] Your books actions and reducers should be configured using `createSlice` from redux toolkit. Kindly refactor this code using `createSlice` instead. You can consider looking at [this article](https://www.softkraft.co/how-to-setup-slices-with-redux-toolkit/) for a general overview of the required setup.

File: src/redux/categories/Categories.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+};
+
+// Action Creators
+const checkStatusAction = () => ({
+  type: checkStatus,
+});
+
+// Export
+export { checkStatusAction };
+export default categoriesReducer;
```
Comment (TedLivist) [inline src/redux/categories/Categories.js]: - [ ] Your categories action and reducer should be configured using `createSlice` from redux toolkit. Kindly refactor this code using `createSlice` instead. You can consider looking at [this article](https://www.softkraft.co/how-to-setup-slices-with-redux-toolkit/) for a general overview of the required setup.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Bigizi,

Good job so far!
✔️ Use of configureStore from redux toolkit
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] Kindly remove all this heading from your README for proper documentation

  image

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — APPROVED** [review]
# Hi @Bigizi,👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🔥🔥💯💯



## Highlights 🔥🔥🔥🔥
- Followed Gitflow ✅
- No Linters error 🤩🔥💯
- Great working on the functionality🔥🔥
- Professional README 🔥✨💯💯


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**